tests/python/states/slater_test.py

```diff
@@ -0,0 +1,70 @@
+"""Test Slater sampler."""
+
+from __future__ import annotations
+
+import math
+
+import numpy as np
+import pytest
+
+import ffsim
+
+
+def _empirical_distribution(bts_matrix, norb, nelec):
+    indices = np.zeros(bts_matrix.shape[0], dtype=int)
+    for i, bts in enumerate(bts_matrix):
+        string = np.array2string(bts, separator="")[1:-1]
+        index = ffsim.strings_to_indices([string], norb, nelec)[0]
+        indices[i] = index
+
+    unique_indices, counts = np.unique(indices, return_counts=True)
+    if isinstance(nelec, tuple):
+        probabilities = np.zeros(math.comb(norb, nelec[0]) * math.comb(norb, nelec[1]))
+    else:
+        probabilities = np.zeros(math.comb(norb, nelec))
```

Comment:
```suggestion
    probabilities = np.zeros(ffsim.dim(norb, nelec))
```
This works for both the spinful and spinless cases.